discovery_engine_core/web-api/Dockerfile.ingestion

```diff
@@ -1,11 +0,0 @@
-FROM debian:bookworm-slim
-
-WORKDIR /service
-
-COPY ./ ./
-
-RUN chmod +x ./ingestion
-
-EXPOSE 3000
```

Comment:
that is:

1. incorrect port
2. is purely informational/metadata and not used by anything

so I removed it